app/controllers/FaciaToolController.scala

```diff
@@ -41,26 +41,41 @@ class FaciaToolController(
         Ok(Json.toJson(configJson)).as("application/json")}}}
 
   def getCollection(collectionId: String) = AccessAPIAuthAction.async { implicit request =>
-
     val collectionPriorities = configAgent.getFrontsPermissionsPriorityByCollectionId(collectionId)
 
     withModifyGroupPermissionForCollections(collectionPriorities, Set()) {
+      val collections = fetchCollections(List(collectionId))
 
-      FaciaToolMetrics.ApiUsageCount.increment()
-      val collection = frontsApi.amazonClient.collection(collectionId).flatMap { collectionJson =>
-        collectionJson.map(json => mediaServiceClient.addThumbnailsToCollection(json, collectionId)) match {
-          case Some(f) => f.map(Some(_))
-          case None => Future.successful(None)
-
-        }
-      }
-      collection.map(c => NoCache {
+      collections.head.map(c => NoCache {
         Ok(Json.toJson(c)).as("application/json")
       })
+    }
+  }
+
+  def getCollections(collectionIdsString: String) = AccessAPIAuthAction.async { implicit request =>
```

Comment:
Can this simply be called `collectionIds`?

Reply:
it's named to disambiguate it from the collectionIds List that follows on line 56, so it needs to be called something else, but perhaps there's a better name?

Ideally we'd be acting on a List[String] to start with -- we discussed binding query params in the route but weren't sure if this was possible, any help welcomed :)

Reply:
Ah I see - this link looks useful: https://stackoverflow.com/questions/36562330/query-parameter-as-list

But may be overkill for what you're trying to do. Worth exploring.

Reply:
Repeating the query param gives you a simple solution given you control the client request.

Reply:
Yes, much prefer to handle query param -- thanks for this.